lisa/tools/dhclient.py

```diff
@@ -69,8 +69,11 @@ def renew(self, interface: Optional[str] = None) -> None:
                 sudo=True,
             )
         else:
-            result = self.run("-r && dhclient", shell=True, sudo=True)
-        if result.exit_code != 0:
-            raise LisaException(
-                f"dhclient renew return non-zero exit code: {result.stdout}"
+            result = self.run(
+                "-r && dhclient",
+                shell=True,
+                sudo=True,
```

Comment:
Remove the unnecessary comma at the end, so all arguments can be in the same line as before.